eclass/distutils-r1.eclass

```diff
@@ -1203,6 +1212,13 @@ distutils-r1_python_compile() {
 		else
 			esetup.py build -j "${jobs}" "${@}"
 		fi
+	elif [[ ${DISTUTILS_USE_PEP517} == maturin ]]; then
+		# auditwheel may attempt to auto-bundle libraries, bug #831171
+		local -x MATURIN_PEP517_ARGS=--skip-auditwheel
+
+		# support cargo.eclass' IUSE=debug if available
+		in_iuse debug && use debug &&
+			MATURIN_PEP517_ARGS+=" --cargo-extra-args=--profile=dev"
```

Comment:
Actually, why are we adding flags for this in the first place? This really feels like adding a flag for `-O0`. This should really be passed via some user variable (`${CARGO_EXTRA_ARGS}`?), not special USE flags.

Reply:
Hmm, cargo.eclass is adding this crap unconditionally? I guess nevermind then.

Reply:
Yeah, having a IUSE that's a no-op is the primary thing that bothers me with it. Also it changes the target directory from target/release to target/debug and it can break ebuilds if a user just pass it -- see old adblock ebuild having a `target/$(usex debug{,} release)`.

Reply:
Oh yeah, for profile there's an additional problem if users try to set it.

passing `--release --profile=dev` = fails

maturin specially removes --release if --profile is in cargo-extra-args

Reply:
Ultimately, not that users should be changing e.g. CMAKE_BUILD_TYPE themselves either and set individual options instead... so I guess these points don't mean much.

Cargo.toml may enable special debug paths with dev profiles, but well.